### ISSUE TITLE
Epochs, regularization, data augmentation

### ISSUE THREAD

**Issue by mansi1710** [NJACKWinterOfCode/Alzheimer-s-Disease-Prediction-using-3D-MRI-scans#4]
Vary epochs, learning rate, other parameters.
Try regularization, data augmentation to handle overfitting
